apps/io-wallet-user-func/src/app/main.ts

```diff
@@ -94,3 +85,16 @@ app.http("getUserByFiscalCode", {
     userRepository: pdvTokenizerClient,
   }),
 });
+
+app.timer("generateEntityConfiguration", {
+  schedule: "0 0 0 * * *",
```

Comment:
Since the lifetime of a JWT by default is set to 1 hour, it is important that the cron is configured in a shorter time interval. https://github.com/pagopa/io-wallet/blob/dfb5e41c1e1fdaf140678d5aa2a84d2728db2c6a/apps/io-wallet-user-func/src/app/config.ts#L150

However, I suggest leaving 24 hours (or a little less) and setting the JWT duration of only the entityConfiguration to 24 hours. You can add the parameter  `jwtDuration` [here](https://github.com/pagopa/io-wallet/blob/dfb5e41c1e1fdaf140678d5aa2a84d2728db2c6a/apps/io-wallet-user-func/src/entity-configuration.ts#L119) like '24h'

Reply:
Because JWTs are also signed for other purposes, such as Wallet Attestations which must have a shorter duration